artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java

```diff
@@ -110,25 +112,27 @@ public Object execute(ActionContext context) throws Exception {
          return null;
       }
 
+      int printingMaxRows = this.maxRows != null ? this.maxRows : DEFAULT_MAX_ROWS;
```

Comment:
this can revert now as you just use maxRows

Reply:
The maxRows is null if the user doesn't set the parameter. I need a way to undestand if the user set the parameter maxRows. I would print a warning only if the user doesn't set the parameter maxRows.

Reply:
but why discriminate? surely just simplify the logic, because if i set 100 but count was 1001 it still be useful